src/dmd/escape.d

```diff
@@ -2430,3 +2447,38 @@ private bool setUnsafeDIP1000(Scope* sc, bool gag, Loc loc, const(char)* msg, Ro
         return false;
     }
 }
+
+/***************************************
+ * Check that taking the address of `v` is `@safe` because of transitive scope.
```

Comment:
I think we should refrain from using "transitive scope" because it causes confusion. Maybe `checkScopeVarAddr` is better?

Reply:
Renamed